<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>982</width>
    <height>457</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>mainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color:lightgray;
}
QPushButton
{
	border:0px;
	background-color: rgb(189, 89, 32,120);
	font:bold;
	color: white;
	padding:10px;
}
QPushButton#afBryd
{
	background-color:red;
}
#afBryd:hover
{
	background-color: rgb(255, 146, 148);
}
#afBryd:pressed
{
	background-color:white;
}
QPushButton#gem
{
	background-color:green;
}
#gem:hover
{
	background-color: rgb(149, 255, 208);
}
#gem:pressed
{
	background-color:white;
}
QPushButton:pressed
{
background-color:red;
}
QPushButton:hover
{
	background-color:lightgray;
	border:5px solid qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.757, fx:0.5, fy:0.5, stop:0 rgba(13, 212, 61, 228), stop:1 rgba(255, 255, 255, 255));
	padding:0px;
}
QPushButton#viewBut
{	
	background-color: transparent;
}
QPushButton:checked
{
	background-color:rgb(175, 255, 26)
}
QWidget#widget
{
	background-color: lightgray;
	border: 0px solid;
}
QMenuBar
{
	background-color: rgb(93, 56, 49);
	color:white;
	width:144px;
}
QMenu
{
	background-color:rgb(93, 56, 49);
	color:white;
}
QMenu::item:selected
{
	background-color:rgb(212, 212, 212);
}
QMenuBar::item
{
	padding:5px;
	margin-right:5px;
}
QMenuBar::item:selected
{
	background-color:lightgray;
}
QMenuBar::item:pressed
{
}
QGroupBox
{
	background-color: transparent;
	border:2px solid white;
	margin-top:10px;
	padding:5px;
}
QGroupBox::title
{
	subcontrol-origin: margin;
    subcontrol-position: top center;
    padding: 0 3px;
	border:1px solid;
	background-color:white;
}
QScrollBar::handle
{
background-color:rgb(81,81,81);
border:2px solid rgb(81,81,81);
border-radius:5px;
}
QSplitter#hSplitter::handle
{
	
	background-color: rgb(189, 89, 32);
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QSplitter" name="hSplitter">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777213</width>
            <height>32</height>
           </size>
          </property>
          <property name="sizeIncrement">
           <size>
            <width>0</width>
            <height>64</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:rgb(189,89,32);
color:rgb(132,66,15);
border-bottom:2px solid black;</string>
          </property>
          <property name="text">
           <string>Directory list</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="myVerticalSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="handleWidth">
           <number>0</number>
          </property>
          <widget class="QWidget" name="horizontalLayoutWidget">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="sizeConstraint">
             <enum>QLayout::SetMaximumSize</enum>
            </property>
            <item>
             <widget class="QWidget" name="widget" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QPushButton" name="clearButt">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="toolTip">
                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Ryd alle dine mapper.&lt;/p&gt;&lt;p&gt;Pas på. Kan ikke fortrydes.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                 <property name="text">
                  <string>Ryd dine valgte mapper</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="countButt">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>Vis størrelse på mapper</string>
                 </property>
                 <property name="flat">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="addBut">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>Tilføj mappe</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="delButt">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>Fjern mappe fra listen</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QTreeWidget" name="dirView">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <column>
               <property name="text">
                <string notr="true">1</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QTreeWidget" name="detailView">
           <column>
            <property name="text">
             <string notr="true">1</string>
            </property>
           </column>
          </widget>
          <widget class="QTreeWidget" name="suffixTree">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="columnCount">
            <number>2</number>
           </property>
           <column>
            <property name="text">
             <string>Suffix</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Count</string>
            </property>
           </column>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="fileInformationView" name="fileInformationBrowserView"/>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="statusLineEdit">
      <property name="styleSheet">
       <string notr="true">border-top:1px solid black;</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>No status</string>
      </property>
      <property name="cursorMoveStyle">
       <enum>Qt::LogicalMoveStyle</enum>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>982</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Menu</string>
    </property>
    <property name="separatorsCollapsible">
     <bool>false</bool>
    </property>
    <addaction name="actionIndstillinger"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuDevelopment">
    <property name="title">
     <string>For developers</string>
    </property>
    <addaction name="actionOpen_current_directory"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuDevelopment"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionIndstillinger">
   <property name="text">
    <string>Indstillinger</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionOpen_current_directory">
   <property name="text">
    <string>Open current directory</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>fileInformationView</class>
   <extends>QTextBrowser</extends>
   <header>fileinformationview.h</header>
  </customwidget>
  <customwidget>
   <class>myVerticalSplitter</class>
   <extends>QSplitter</extends>
   <header>myverticalsplitter.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>fileInformationBrowserView</tabstop>
  <tabstop>statusLineEdit</tabstop>
  <tabstop>suffixTree</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
